(bug): Controller error during pv discovery

When attempting PV discovery, I am seeing the UI hang. Looked at the pod list & the controller was entering into a crash loop. Other useful info that may help in debugging: I am running the UI on my 4 cluster, but I am attempting to create a migplan to migrate a 3 cluster to a separate 4 cluster (not the host cluster I am running the UI from). 

```
/go/src/github.com/konveyor/mig-controller/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88
/usr/local/go/src/runtime/asm_amd64.s:2361
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x108 pc=0x13717fe]

goroutine 265 [running]:
github.com/konveyor/mig-controller/vendor/k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/konveyor/mig-controller/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:58 +0x107
panic(0x18edfa0, 0x2b24940)
	/usr/local/go/src/runtime/panic.go:502 +0x229
github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1.(*MigCluster).BuildRestConfig(0xc42359cea0, 0x1d50a60, 0xc42487c000, 0x1d50a60, 0xc42487c000, 0x0)
	/go/src/github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1/migcluster_types.go:164 +0x12e

```

--- a/pkg/controller/migplan/validation.go
+++ b/pkg/controller/migplan/validation.go
@@ -900,7 +900,7 @@ func (r *NfsValidation) init(client k8sclient.Client) error {
 			log.Trace(err)
 			return err
 		}
-		r.restCfg, err = r.cluster.BuildRestConfig(r.client)
+		r.restCfg, err = r.cluster.BuildRestConfig(client)
 		if err != nil {
 			log.Trace(err)
 			return err

